devtools/server/actors/replay/module.js

```diff
@@ -265,7 +265,7 @@ function setSourceMap({
     targetMapURLHash: makeAPIHash(sourceMapURL),
 
     // Attempt to be more specific by matching on the script's URL and content.
-    targetContentHash: typeof objectText === "string" ? makeAPIHash(objectText) : undefined,
+    targetContentHash: objectHash,
```

Comment:
So the final patch integrates xxhash instead (it's apparently faster).  Should we use 'xxh128:' or something?

Reply:
I think I initially missed that we were still considering changing the hash to something other than SHA256. We'll need backend support for the new type of hash, which isn't a trivial amount of work. I'm very tempted to say that that might be only slightly less work than supporting `sourceId` in the API instead. The more of think about this, the less I think adding a new type of hash is a worth the effort relative to the other option.

Reply:
Ok I've switched it back to SHA256 on canonicalised UTF8 text as before.  I'll double-check the hashes are the same as it would have been in prior code, but we should be good to go on this front.